test/integration/services/patient-quick-search.js

```diff
@@ -213,6 +215,29 @@ context('Patient Quick Search', function() {
       .find('.js-picklist-item')
       .should('have.length', 10);
 
+    cy
+      .get('@searchModal')
+      .find('.js-picklist-item')
+      .first()
+      .find('.patient-search__picklist-item-meta')
+      .eq(1)
+      .should('contain', '123-45-6789');
+
+    cy.window().then(win => {
+      cy
+        .get('@searchModal')
+        .find('.js-picklist-item')
+        .first()
+        .find('.patient-search__picklist-item-meta')
+        .last()
+        .then($el => {
+          const before = win.getComputedStyle($el[0], '::before');
+          const beforeContent = before.getPropertyValue('content');
+
+          expect(beforeContent).to.equal('"–"');
+        });
+    });
+
```

Comment:
In this test I display two patient identifiers in the search results: 

* `SSN` that contains a value.
* `MRN` that contains an empty string value.

For the empty `MRN` value, I test that the `::before` selector contains the correct `"-"` string for its `content` CSS property. Found a way to do that in Cypress from [here](https://github.com/bahmutov/cypress-examples/blob/master/docs/recipes/pseudo-selectors.md#before-content-css-selector-before).

Reply:
This is interesting but quite complicated for something we should maybe test for in more places.  I either think we test for empty and let the styles go we should make a custom command for this such that the API ends up being:
```js
      cy
        .get('@searchModal')
        .find('.js-picklist-item')
        .first()
        .find('.patient-search__picklist-item-meta')
        .hasBeforeContent('-');
```
It'd have to use the previous context and maybe `within` like the example you referenced.

Reply:
For future reference, we decided to add the `hasBeforeContent()` Cypress command via this commit: https://github.com/RoundingWell/care-ops-frontend/pull/784/commits/4d1aac900a0c7f9e6df53faf7cdb01f4a4d89e47.